### PR TITLE
Fixed some issues with the OAuth signature computation

### PR DESCRIPTION
**Pull request by mbaynton**
Fixes some things that could get in the way of successful exchanges with Mollom. Good chance this addresses https://github.com/Mollom/MollomPHP/issues/13 and/or https://github.com/Mollom/MollomPHP/issues/8.

### CODE REVIEW

File: mollom.class.inc [hunk 1]
```diff
@@ -680,7 +680,7 @@ abstract class Mollom {
     // to query parameters.
     if ($this->oAuthStrategy == 'header') {
       foreach ($oauth as $key => $value) {
-        $oauth[$key] = $key . '="' . self::rawurlencode($value) . '"';
+        $oauth[$key] = self::rawurlencode($key) . '="' . self::rawurlencode($value) . '"';
```
Comment (lkhaas) [inline mollom.class.inc]: So now this needs to be converted to plain rawurlencode per your other pr.

File: mollom.class.inc [hunk 2]
```diff
@@ -832,7 +832,7 @@ abstract class Mollom {
         $key = rawurlencode($parent);
       }
       else {
-        $key = ($parent ? $parent . '[' . rawurlencode($key) . ']' : rawurlencode($key));
+        $key = ($parent ? $parent . rawurlencode('[' . $key . ']') : rawurlencode($key));
```
Comment (lkhaas) [inline mollom.class.inc]: Ditto here.

**Comment by lkhaas** [comment]
Actually - went ahead and made the changes and committed here: https://github.com/Mollom/MollomPHP/commit/c8f125aa71225f0d55f3882d04f84716956ade03

Thanks again for the contribution!

**Comment by mbaynton** [comment]
Right, thanks for finishing it up. Glad to see these pr's are both in.

**Pull request closed without merging**